crates/networking/discv5/Cargo.toml

```
@@ -10,3 +10,11 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+# ream dependencies
+ream-network-types = { workspace = true }
+
+discv5 = { workspace = true }
+futures = { workspace = true }
+libp2p-identity = { workspace = true }
+libp2p-mplex = { workspace = true }
+libp2p = { version = "0.54.1", default-features = false, features = ["identify", "yamux", "noise", "dns", "tcp", "tokio", "plaintext", "secp256k1", "macros", "ecdsa", "metrics", "quic", "upnp", "gossipsub", "ping"] }
```

Duplicated declaration with `Cargo.toml` at root.
```suggestion
libp2p = { workspace = true }
```